### PR TITLE
Adding Syslog Workbook

### PR DESCRIPTION
**Pull request by damendo**
## Summary 

Container insights is adding the capability to collect syslog from AKS Linux node. This out-of-box workbook will help customers view syslogs easily.

Screenshot_1
Screenshot_2

## PR Checklist

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes
* [x] ensure your template has a corresponding gallery entry in the gallery folder
* [x] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [x] ensure all steps have meaningful names
* [x] ensure all parameters and grid columns have display names set so they can be localized
* [x] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [x] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/AKS/Syslog/Syslog.workbook [hunk 1]
```diff
@@ -0,0 +1,568 @@
...
+          {
+            "id": "0698db8c-7a3a-4aec-bfb2-eb59942b0375",
+            "version": "KqlParameterItem/1.0",
+            "name": "Workspace",
+            "type": 5,
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "resources\r\n| where type =~ '{ResourceType}'\r\n| where name == \"{Resource:Name}\"\r\n| extend workspaceResourceId = tostring(parse_json(tolower(properties)).addonprofiles.omsagent.config.loganalyticsworkspaceresourceid)\r\n| project workspaceResourceId ",
```
Comment (zishinchua) [inline Workbooks/AKS/Syslog/Syslog.workbook:112]: where name **==** \"{Resource:Name} should be where name =~ for case insensitive comparison

Comment (damendo) [reply Workbooks/AKS/Syslog/Syslog.workbook]: Update

File: gallery/container-insights/Microsoft.ContainerService-managedClusters.json [hunk 2]
```diff
@@ -132,8 +132,15 @@
 					"description": "Provide monitoring for services in Open Service Mesh",
 					"name": "OSM monitoring",
 					"isPreview": true
+				},
+				{
+					"id": "Workbooks/AKS/Syslog",
+					"author": "Microsoft",
+					"description": "Provide summary and visualisations for Syslog",
+					"name": "Syslog",
+					"isPreview": true
```
Comment (zishinchua) [inline gallery/container-insights/Microsoft.ContainerService-managedClusters.json:141]: I don't think isPreview is needed here, can be removed

Comment (damendo) [reply gallery/container-insights/Microsoft.ContainerService-managedClusters.json]: Done

**Comment by damendo** [comment]
#sign-off

**Review by zishinchua — APPROVED** [review]

**Pull request merged**